src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud/help-directConnection.html

```diff
@@ -1,6 +1,6 @@
 <div>
   <p>
-  With this option is is possible to connect directly to the TCP agent listener port.
+  With this option is possible to connect directly to the TCP agent listener port.
```

Comment:
Good catch that there was a typo here, but I think you fixed the wrong mistake.

```suggestion
  With this option it is possible to connect directly to the TCP agent listener port.
```